### ISSUE TITLE
Add validation + evaluation on the test set

### ISSUE THREAD

**Issue by EgorLakomkin** [Cyberlander/LSTMIMDB#2]
In paper 15% of the training set is used to monitor training progress.

It would be useful to have a routine that evaluates several things on the held out data:
 - accuracy
 - average % of the input needed to be processed

We can call this routine on validation and test data to report the performance and compare with the original paper
